OIPA/iati/models.py

```diff
@@ -53,6 +53,7 @@ class Activity(models.Model):
     hierarchy = models.SmallIntegerField(choices=hierarchy_choices, default=1, null=True)
     # last_updated_datetime = models.CharField(max_length=100, default="")
     # value_date = models.DateField(null=True)
+    last_updated = models.DateTimeField(null=True, auto_now=True)
```

Comment:
is used for search , but can be removed since we do an update with a listener, but then I have to change the search index